content/docs/getting-started.md

```diff
@@ -9,17 +9,21 @@ weight: 10
 
 ## Building the dex binary
 
-Dex requires a Go installation and a GOPATH configured. For setting up a Go workspace, refer to the [official documentation][go-setup]. Clone it down the correct place, and simply type `make` to compile the dex binary.
```

Comment:
We may not want to remove the reference to the `GOPATH` here as we do reference it later on. (and I don't think it is setup by default if you install Go)

Reply:
Gopath is not mandatory after migrating to go modules. If it is confusing, I can completely remove it.

Reply:
I'm able to run the examples w/o setting the $GOPATH, and if it's not mandatory, it may be best to remove it as it is extra detail that makes the getting started more complicated.

Reply:
Ok, I have no objections.